package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "tslint -p . -c tslint.json -t verbose",
     "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
     "generate:pagopa-api-definitions": "rimraf definitions/pagopa && mkdir -p definitions/pagopa && gen-api-models --api-spec https://acardste.vaservices.eu:1443/pp-restapi/v2/api-docs --out-dir ./definitions/pagopa",
-    "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.41/api_proxy.yaml --out-dir ./definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.41/api_public.yaml --out-dir ./definitions/backend",
+    "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.42/api_proxy.yaml --out-dir ./definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.41/api_public.yaml --out-dir ./definitions/backend",
```

Comment:
it looks like you've missed one 0.41

Reply:
Not a big problem the api_public.yaml file is not changed in v0.0.42 but i am going to update that line with another PR.